1-getting-started/Translations/README.ja.md

```diff
@@ -0,0 +1,16 @@
+# IoTことはじめ
+
+本カリキュラムのこのセクションでは、モノのインターネット (Internet of Things) の概要と基本的なコンセプトを学びます。また、クラウドに接続する初めての 'Hello World' IoT プロジェクトに取り組みます。このプロジェクトでは、センサーによって計測された光量が下がると点灯する常夜灯を作ります。
+
+![光量に応じて点いたり消えたりする、WIOに繋げられたLED](../../images/wio-running-assignment-1-1.gif)
+
+## トピック
+
+1. [IoT入門](lessons/1-introduction-to-iot/README.md)
```

Comment:
Are these links correct? Don't they need to point up one folder?

Reply:
Oh, true, thank you for pointing out that. I have just fixed the link and verified that the link works correctly in my forked repository.